sklearn/metrics/tests/test_score_objects.py

```diff
@@ -459,18 +459,38 @@ def test_supervised_cluster_scorers():
         assert_almost_equal(score1, score2)
 
 
-@ignore_warnings
 def test_raises_on_score_list():
     # Test that when a list of scores is returned, we raise proper errors.
     X, y = make_blobs(random_state=0)
     f1_scorer_no_average = make_scorer(f1_score, average=None)
     clf = DecisionTreeClassifier()
-    with pytest.raises(ValueError):
+    score_kwargs = {'error_score': np.nan}
+
+    # since we're using a list of scores, our error will be the following
+    error_message = "ValueError: scoring must return a number"
+
+    # the warning message we're expecting to see
+    warning_message = ("Scoring failed. The score on this train-test "
+                       "partition for these parameters will be set to %f. "
+                       "Details: \n" % score_kwargs['error_score'])
+
+    with pytest.warns(UserWarning, match=warning_message) as record:
         cross_val_score(clf, X, y, scoring=f1_scorer_no_average)
+    assert len(record) > 0
+    for item in record:
+        assert 'Traceback (most recent call last):\n' in str(item.message)
+        split = str(item.message).splitlines()
+        assert error_message in split[-1]
```

Comment:
i think we usually test for this, I feel like it's too strict of a test

Reply:
@adrinjalali Removed and simplified, can you please review?